cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -171,35 +171,29 @@ def _cleanup():
         target_provider=provider,
         vm_list=[vm_obj])
 
-    migration_plan.wait_for_state("Started")
+    migration_plan.wait_for_state("In_Progress")
     request_details_list = migration_plan.get_plan_vm_list(wait_for_migration=False)
     vm_detail = request_details_list.read()[0]
+    request_details_list.cancel_migration(vm_detail, confirmed=True)  # Cancel migration
 
-    def _get_plan_status_and_cancel():
-        migration_plan_in_progress_tracker = []
-        clock_reading1 = request_details_list.get_clock(vm_detail)
-        time.sleep(1)  # wait 1 sec to see if clock is ticking
-        if request_details_list.progress_percent(vm_detail) > 20:
-            request_details_list.cancel_migration(vm_detail, confirmed=True)
-        clock_reading2 = request_details_list.get_clock(vm_detail)
-        migration_plan_in_progress_tracker.append(
-            request_details_list.is_in_progress(vm_detail) and (clock_reading1 < clock_reading2))
-        return not any(migration_plan_in_progress_tracker)
+    def get_plan_status_and_cancel():
+        return not request_details_list.is_in_progress(vm_detail)
 
     wait_for(
-        func=_get_plan_status_and_cancel,
+        func=get_plan_status_and_cancel,
         delay=10,
-        num_sec=3600,
+        num_sec=600,
         message="migration plan is in progress, be patient please")
 
     soft_assert(request_details_list.is_cancelled(vm_detail))
     soft_assert(request_details_list.progress_percent(vm_detail) < 100.0 or
                 "Virtual machine migrated" not in request_details_list.get_message_text(vm_detail))
 
+    vm_on_dest = provider.mgmt.find_vms(name=vm_obj.name)
     # Test1: Check if instance is on openstack/rhevm provider
-    soft_assert(not provider.mgmt.find_vms(name=vm_obj.name))
+    soft_assert(not vm_on_dest)
```

Comment:
I'm suspect about this check.
why soft_assert?
are we expect that VM on `Openstack/rhevm` provider? if yes, why negation in an assertion?

Reply:
No, we are not expecting VM at destination.

Reply:
The TC gets failed if VM gets migrated to Destination, However for OSP the port and volumes gets attached to the VM once it gets migrated , So for OSP, we need to do additional check for port and volumes if VM gets migrated to Destination.